src/elements/OcModal.vue

```diff
@@ -217,6 +217,12 @@ export default {
     }
   },
 
+  mounted() {
+    this.$nextTick(() => {
+      this.$refs.$_ocModal.focus()
```

Comment:
does this focus on the input field ?

would it also focus on the cancel button if there is no input field ? (if that makes sense)

Reply:
Nope, it focuses the modal itself because of accessibility as described in here https://github.com/owncloud/phoenix/issues/2263#issuecomment-543516672